sys/auto_init/saul/auto_init_tmp006.c

```diff
@@ -60,6 +60,10 @@ void auto_init_tmp006(void)
             LOG_ERROR("[auto_init_saul] error set active tmp006 #%u\n", i);
             continue;
         }
+        if (tmp006_set_standby(&tmp006_devs[i]) != TMP006_OK) {
+            LOG_ERROR("[auto_init_saul] error set active tmp006 #%u\n", i);
```

Comment:
copy-paste typo: should be standby instead of active

Reply:
My bad :( Corrected and Thanks!